packages/blade/src/components/Drawer/_decisions/decisions.md

```diff
@@ -0,0 +1,147 @@
+# Drawer
+
+A drawer is a panel that slides in mostly from right side of the screen over the existing content in the viewport. It helps in providing additional details or context and can also be used to promote product features or new products.
+
+<img width="500px" src="./2024-02-07-20-04-07.png" alt="Drawer Figma Skeleton">
+
+## Design
+
+- [Drawer - Figma Design](https://www.figma.com/file/jubmQL9Z8V7881ayUD95ps/Blade-DSL?node-id=78667%3A66663&mode=dev)
```

Comment:
Does the drawer have a fixed width? Will the width change based on the content? If it is fixed width, why do we not have size options?

Reply:
It currently has a fixed max-width of 420px and inside that it takes width based on the content.

Let me check with design again once if we want to go with max-width or defined size because in Modal I see its defined size

Reply:
Just checked with RK. It is fixed width of 420px on desktop always. We will only support one size right now

Reply:
> fixed width of 420px
Max width would be 420px right?

Reply:
nope. Drawer is always 420px irrespective of the content.